scalableminds/webknossos
# organization-specific newUserMailingList and overTimeMailingList

### URL of deployed dev instance (used for testing):
- https://organizationmail.webknossos.xyz

### Steps to test:
- set up Mailer
- set the two new field in the db to real email adresses
- check if you get something

### Issues:
- fixes #2932

------
